macros/manual_change.ngc

```diff
@@ -58,7 +59,6 @@ O500 endif
 
 G90
 G53 G0 Z[#<_ini[CHANGE_POSITION]Z>]
-G53 G0 X[#<_ini[CHANGE_POSITION]X>] Y[#<_ini[CHANGE_POSITION]Y>]
```

Comment:
Again, this wasn't strictly a 2.8 change - however there was no reason to return to the X/Y change position after a tool change that I could see.